src/web/components/SlotBodyEnd.tsx

```diff
@@ -20,10 +25,25 @@ const sendOphanInsertEvent = (): void => {
             name: 'remote_epic_test',
             variant: 'api',
         },
-        action: 'INSERT',
+        action,
     };
 
-    window.guardian.ophan.record({ componentEvent });
+    const key = JSON.stringify(componentEvent);
+    if (!ophanEventRegister[key]) {
+        ophanEventRegister[key] = true;
+        console.log(`Sending ${action} event to Ophan`);
+        window.guardian.ophan.record({ componentEvent });
```

Comment:
Would a try catch here be the best way to capture the failure path? Not really sure how ophan works here tbh

Reply:
Good question! I've not dug too much into the inner workings of this data getting to Ophan so I'm not sure either. I'll take a look though.